### ISSUE TITLE
With vagrant-libvirt < 0.0.45, vagrant up fails with "The following settings shouldn't exist: qemu_use_session"

### ISSUE THREAD

**Issue by sebastian-philipp** [SUSE/sesdev#43]
```
vagrant up
Bringing machine 'admin' up with 'libvirt' provider...
Bringing machine 'node1' up with 'libvirt' provider...
Bringing machine 'node2' up with 'libvirt' provider...
Command '['vagrant', 'up']' failed: ret=1 stderr:
b"==> node2: An error occurred. The error will be shown after all tasks complete.
==> node1: An error occurred. The error will be shown after all tasks complete.
==> admin: An error occurred. The error will be shown after all tasks complete.
An error occurred while executing multiple actions in parallel.
Any errors that occurred are shown below.

An error occurred while executing the action on the 'admin'
machine. Please handle this error then try again:

There are errors in the configuration of this machine. Please fix
the following errors and try again:

Libvirt Provider:
* The following settings shouldn't exist: qemu_use_session



An error occurred while executing the action on the 'node1'
machine. Please handle this error then try again:

There are errors in the configuration of this machine. Please fix
the following errors and try again:

Libvirt Provider:
* The following settings shouldn't exist: qemu_use_session



An error occurred while executing the action on the 'node2'
machine. Please handle this error then try again:

There are errors in the configuration of this machine. Please fix
the following errors and try again:

Libvirt Provider:
* The following settings shouldn't exist: qemu_use_session
```

```
$ vagrant plugin list
vagrant-libvirt (0.0.43, system)
```
So - this is supposed to work in newer versions of `vagrant-libvirt`?

**Comment by ricardoasmarques** [comment]
I'm using `0.0.45` and it's working:
```
$ vagrant plugin list
vagrant-libvirt (0.0.45, system)
```

**Comment by sebastian-philipp** [comment]
as upgrading `vagrant-libvirt` is painful, I'll likely leave it as it is and stay with the bug for now.

**Comment by smithfarm** [comment]
Fixed by vagrant 0.0.45 which is available in Virtualization:vagrant
